TwilioVerifySDK/TwilioVerify/Sources/API/FactorAPIClient.swift

```diff
@@ -149,6 +149,10 @@ private extension FactorAPIClient {
     body.append(contentsOf: createFactorPayload.config.map { config in
       Parameter(name: "\(Constants.configKey).\(config.key)", value: config.value)
     })
+    let encoder = JSONEncoder()
+    if let metadata = createFactorPayload.metadata, let jsonData = try? encoder.encode(metadata), let jsonString = String(data: jsonData, encoding: .utf8) {
+      body.append(Parameter(name: Constants.metadataKey, value: jsonString))
```

Comment:
According to Android, in case `metadata` is null the factor body will contain {'metadata': 'null'}, null as string. But in this case if metadata is null no value is sent, how should it be?

Reply:
It should be null, fixing Android implementation

Reply:
https://github.com/twilio/twilio-verify-android/pull/212/files#diff-f5bb222afe92724c61bf971d4cf74f6a6483d4ce5283f8be74a9ba9a0731dd96R258